### ISSUE TITLE
NTR, pulmo: Elevated level of neutrophil granulocytes

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5562]
**Preferred term label:**
Elevated level of neutrophil granulocytes

**Synonyms**
Elevated level of neutrophils in broncho-alveolar lavage fluid

**Definition (free text, please give PubMed ID)**
Usually, Neutrophils make up less than 3% of all cells found in the broncho-alveloar lavage fluid. In children, standard value of neutrophils is higher depending on their age (children under the age of 5 show a maximum value of 10%). This elevated cell proportion is a sign for acute and chronic infections (HP:0012387, HP:0006538) and can be associated to specific diseases. 
(Costabel ISBN0412792702; PMID: 3396403; PMID: 10678650)

**Parent term (use hpo.jax.org/app)**
Abnormal cellular composition of bronchoalveolar fluid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:182098, OMIM: 608710, OMIM: 178500, ORPHA:70578

**Your nano-attribution (ORCID)**
crapp, @mgriese, @katrinknoflach

**Comment by drseb** [comment]
Please separate "abnormal levels" from "increased levels" if possible. Thanks! Sebastian

**Comment by GrieseAG** [comment]
Preferred term label:
Elevated level of neutrophil granulocytes

**Comment by pnrobinson** [comment]
adding with label Elevated bronchoalveolar lavage fluid neutrophil proportion

**Comment by pnrobinson** [comment]
added as HP:0032977
